plugins/custom-dashboard/README.md

```diff
@@ -0,0 +1,35 @@
+# Custom Dashboard Plugin
+
+A plugin that enables custom dashboards in the FiftyOne app.
```

Comment:
It's fine to say "custom dashboard" here, but I think the functional nomenclature should be just "dashboard".